sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -494,8 +502,8 @@
         "parameters": [
           {
             "name": "id",
+            "description": "The device Id.",
```

Comment:
This is worded as below, in the other APIs:
The unique identifier of the device to xxx.

Reply:
Is that a common way of describing this?

Reply:
It is, yes